### DISCUSSION TITLE
Issues tab not present

### DISCUSSION THREAD

**Discussion by Lork88** [And3rsL/Deebotozmo#117] [General]
Hello, 
why in this repository the issues tab is not present. I'm pretty sure that 1 week ago it existed and I created a new issue..
Thankyou

**Comment by mrbungle64** [comment]
@Lork88 

The owner of this repo is currently not active.
I would recommend you to use [this](https://github.com/DeebotUniverse/client.py) library. It's a fork of the Deebotozmo library.

**Reply by Lork88** [reply]
Thankyou mrbungle. I've tried right now deebot-client but seems to me that lacks of a cli support. Unfortunately my knoledge is not enough to write custom .py.... But I'm wrong or a is it true that there isn't an equivalent of "getrooms"?

**Reply by mrbungle64** [reply]
That's correct. The Client Library for Deebot Vacuums has currently no cli support and until someone is found, who is implementing and maintaining the cli, no cli support will be added.
I didn't know that your issue was related to the cli integration...

**Reply by Lork88** [reply]
No no it's ok. My issue was related to an asyncio error performing "getrooms" and "spotarea". I don't think it's due to my ozmo 920 but to the code.
